client-ui/src/components/pages/auth/commonAuth.css

```
@@ -0,0 +1,18 @@
+body{
```

Use this formatting in all css rules
```suggestion
body {
```